backend/src/main/resources/db/migration/V1.22.0__NOTIF-190_cronjob_timestamp.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE cronjob_run (
+    id UUID NOT NULL,
+    last_run TIMESTAMP
+) WITH (OIDS=FALSE);
+
+INSERT INTO cronjob_run (id, last_run) VALUES
+('3db0955b-751b-48cd-b531-c1d81596d133', NOW());
```

Comment:
NOW() is probably wrong. Does anyone have a preference what I should insert here?